runners/pom.xml

```diff
@@ -55,6 +55,13 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-surefire-plugin</artifactId>
+              <dependencies>
+                <dependency>
+                  <groupId>org.apache.beam</groupId>
+                  <artifactId>beam-sdks-java-core</artifactId>
+                  <type>test-jar</type>
```

Comment:
What is the diff to the pom for this? I guess I never even tried just changing this to a `classifier`. Is that known not to work?